Lib-Open5e/o5e_Util_inferProficiencyAndBonus.xml

```diff
@@ -0,0 +1,39 @@
+<net.rptools.maptool.model.MacroButtonProperties>
+  <macroUUID>4ea5933b-fea8-4dc5-829b-fbc4236c80ca</macroUUID>
+  <saveLocation>Token</saveLocation>
+  <index>20</index>
+  <colorKey>default</colorKey>
+  <hotKey>None</hotKey>
+  <label>o5e_Util_inferProficiencyAndBonus</label>
+  <group>Open 5e - Utility</group>
+  <sortby/>
+  <autoExecute>true</autoExecute>
+  <includeLabel>false</includeLabel>
+  <applyToTokens>false</applyToTokens>
+  <fontColorKey>black</fontColorKey>
+  <fontSize>1.00em</fontSize>
+  <minWidth/>
+  <maxWidth/>
+  <allowPlayerEdits>true</allowPlayerEdits>
+  <toolTip>Given a static bonus value and a base proficiency value, infer the&lt;br&gt; proficiency multiplier (if any) and assume the remainder is a bonus.&lt;br&gt;
+&lt;br&gt;
+This makes a few assumptions:&lt;br&gt;
+- it is only ever 0, 0.5, 1, or 2 the proficiency&lt;br&gt;
+- the bonus is never greater than the proficiency product&lt;br&gt;
```

Comment:
"... until the proficiency is calculated as 2, in which case the entire remainder is considered to be the bonus."